proposals/btco-vc/btco_vc_proposal.md

```diff
@@ -0,0 +1,96 @@
+
+- Enable creation of tamper-proof verifiable credentials directly on Bitcoin
+- Provide a standardized approach for digital artifact verification
+- Leverage Bitcoin's security and decentralization for credential issuance
+- Create interoperable standards for credential verification
+- Address privacy considerations in public blockchain deployments
+- Utilize inscriptions' content-addressing capabilities for robust DID Linked Resources
+
+## Coordination/Correspondence
+
+- Bi-weekly updates on progress and blockers
+- Monthly community calls for feedback and discussion
+- GitHub discussions for technical proposals
+- Regular documentation updates
+
+## Prior art
+
+Several key technologies and specifications inform this proposal:
+
+1. [Bitcoin Ordinal Theory & Inscriptions Protocol](https://docs.ordinals.com/) - Casey Rodarmor. The foundational protocol that enables inscribing arbitrary data onto individual satoshis on Bitcoin.
+
+2. [BRC-20](https://domo-2.gitbook.io/brc-20-experiment/) - Domo. An experimental token standard built on Bitcoin Ordinals that demonstrated the viability of implementing standards and protocols using inscriptions.
+
+3. [Verifiable Credentials Data Model 2.0](https://www.w3.org/TR/vc-data-model-2.0/) - Manu Sporny, Dave Longley, David Chadwick, et al. The W3C standard that defines the data model and formats for verifiable credentials.
+
+4. [DID Core Specification](https://www.w3.org/TR/did-core/) - Manu Sporny, Amy Guy, Markus Sabadello, Drummond Reed, et al. The W3C specification that establishes the foundation for decentralized identifiers and their resolution.
+
+5. [DID Linked Resources](https://w3c-ccg.github.io/DID-Linked-Resources/) - Alex Tweeddale. The DIF specification for associating verifiable resources with DIDs in a tamper-evident way.
+
+6. [`did:btco` Method Specification](https://github.com/ordinalsreserve/did-btco) - Brian Richter. A DID method that enables associating DIDs with Bitcoin using Ordinal Theory & Inscriptions.
+
+## Unresolved Questions
+
+1. Scalability Considerations:
+   - What are the optimal inscription sizes for different credential types?
+   - How to handle credential revocation on Bitcoin?
+   - What are the cost implications of different deployment approaches?
+
+2. Privacy Concerns:
+   - How to balance public verifiability with privacy requirements?
+   - What data should be stored on-chain vs off-chain?
+   - How to implement selective disclosure?
+
+3. Technical Implementation:
+   - Best practices for key management in the did:btco method
+   - Approach to handling credential updates and versioning
+   - Integration with existing Bitcoin wallets and infrastructure
+
+4. Deployment Platform:
+   - Evaluation criteria for selecting optimal Bitcoin L2 or sidechain
+   - Trade-offs between different scaling solutions
+   - Integration requirements for Fractal Bitcoin deployment
+
+## Outputs
```

Comment:
I would suggest anchoring on a specific output for DIF Labs. I.e The Technical Specification. Of course, additional outputs are welcome, but my suggestion would be to focus on a single artifact as part of the proposal.